README.md

```diff
@@ -5,7 +5,7 @@ Random Wallpapers for Gnome 3 is a gnome-shell extension that fetches a random w
 
 Install and try the extension at [extensions.gnome.org](https://extensions.gnome.org/extension/1040/random-wallpaper/).
 
-![Screenshot](/assets/screenshot.png)
+![Screenshot](./assets/screenshot.png)
```

Comment:
This does not seem to be necessary?

Reply:
For GitHub Markdown — no, for others, like IDE — yes, see example from WebStorm:

![image](https://user-images.githubusercontent.com/1087411/132013406-6c579f4c-b8ab-403d-bd65-d1cb8095fe4b.png)

Reply:
Thanks for clarification :)